core/src/main/resources/jenkins/model/Jenkins/EnforceSlaveAgentPortAdministrativeMonitor/message.properties

```diff
@@ -1 +1,2 @@
-description=Slave Agent Port has been manually changed. Its value will be reset to {0} on restart.
+description=Slave Agent Port has been manually changed. Its value will be reset to {0,number,#} on restart.
```

Comment:
Should probably mention the system property that sets this, otherwise there's no reference to the reason for this or action to change this behavior.